### PR TITLE
Task 212 - 234   Phase 2

### PR DESCRIPTION
**Pull request by vijaykumawat256**

### CODE REVIEW

**Comment by Ravsehajsinghpuri** [comment]
@vijaykumawat256 1) Can you add the original tasks in this PR. 2) Can you rename the task variants as taskxxx.1_yyyy, taskxxx.2_yyyy, taskxxx.3_yyyy and so on. (For e.g. task547.1_spl_translation_entk_en.json, task547.2_spl_translation_entk_en.json, task547.3_spl_translation_entk_en.json and so on). Do this for all the tasks in this PR 3) Please resolve the conflicts as well

**Comment by vijaykumawat256** [comment]
I have made changes as per the requirement. Please check and let me know if there is any issue.

File: tasks/README.md [hunk 1]
```diff
@@ -1,9 +1,153 @@
-## Tasks List 
+## tasks List 
 
 This directory contains the tasks that are part of this benchmark. 
 
 
 Name | Summary | Category
 ---- | ----------- | --------
-`task1001_squad1.1_question_generation` | Generating guestions (based on SQuAD 1.1) | Question Generation
```
Comment (Ravsehajsinghpuri) [inline tasks/README.md:8]: This task needs to be added

File: tasks/README.md [hunk 2]
```diff
@@ -1,9 +1,153 @@
-## Tasks List 
+## tasks List 
 
 This directory contains the tasks that are part of this benchmark. 
 
 
 Name | Summary | Category
 ---- | ----------- | --------
-`task1001_squad1.1_question_generation` | Generating guestions (based on SQuAD 1.1) | Question Generation  
-`task1002_squad1.1_answer_generation` | Generating answers to SQuAD 1.1 questions | Answer Generation
```
Comment (Ravsehajsinghpuri) [inline tasks/README.md:9]: This task needs be added as well

Comment (vijaykumawat256) [reply tasks/README.md:9]: I have added the required task. Please check now.

**Pull request merged**